pol-core/pol/network/pktboth.h

```diff
@@ -150,9 +151,9 @@ struct PKTBI_3B
     u8 layer;
     u32 item_serial;
     u16 number_bought;
-  } items[1]; /* Dunno how many it should be */
+  } items[( MAXBUFFER - 8 ) / 7];
```

Comment:
just to be sure: is maxbuffer also the maximum msglen? I guess otherwise we would have problems already

Reply:
It's smaller then 0xffff, but somewhere during pktreveive checked.